### PR TITLE
refactor(proxy): remove tls from app

### PR DESCRIPTION
**Pull request by rsdmike**

### CODE REVIEW

File: src/server/webserver.ts [hunk 1]
```diff
@@ -351,38 +275,15 @@ export class WebServer {
...
-        this.sessionParser(request, {}, () => {
-          if (!this.config.auth_enabled || (request.session && request.session.loggedin === true)) { // Validate if the user session is active and valid. TODO: Add user validation if needed
-            this.handleUpgrade(request, socket, head)
-          } else { // Auth failed
-            log.error('WebSocket authentication failed. Closing connection...')
-            socket.write('HTTP/1.1 401 Unauthorized\r\n\r\n')
-            socket.destroy()
-          }
-        })
+        console.log('UPGRADING')
```
Comment (brianosburn-intel) [inline src/server/webserver.ts:285]: can this be changed to the logger?

**Review by brianosburn-intel — APPROVED** [review]
looks good

File: src/server/webserver.ts (hunk 1 above)
Comment (rsdmike) [reply src/server/webserver.ts:285]: imma just delete that

**Comment by rsdmike** [comment]
Snyk is broken on this PR -- regenerating with #154

**Pull request closed without merging**